### ISSUE TITLE
[feature] Custom virtualenv name

### ISSUE THREAD

**Issue by uilianries** [conan-io/conan#4557]
Hi!

This request comes from [stack overflow](https://stackoverflow.com/questions/54642935/how-to-change-the-name-of-virtual-environment-in-conan).

Since we use hard-coded names, like "conanenv", no one is able generate multiple virtualenvs at same place.

We could customize the name by env var, since we don't have extended options for generators.

Regards!

- [x] I've read the [CONTRIBUTING guide](https://github.com/conan-io/conan/blob/develop/.github/CONTRIBUTING.md).
- [ ] I've specified the Conan version, operating system version and any tool that can be relevant.
- [ ] I've explained the steps to reproduce the error or the motivation/use case of the question/suggestion.

**Comment by memsharded** [comment]
It doesn't seem it is about the filename, but about the displayed name in the terminal, at least from what I understand from https://stackoverflow.com/questions/54642935/how-to-change-the-name-of-virtual-environment-in-conan

**Comment by uilianries** [comment]
That's right. Since the generated files are `active.sh` and `deactivate.sh`. Anyway, should we provide such feature?

**Comment by memsharded** [comment]
The only way it could be possible is resurrecting the "parameterized generators" that @lasote did a proposal some time ago, but was closed because it was not clear the use cases.

**Comment by uilianries** [comment]
hmm so I think it's a lot of effort for this feature
